artifacts/Music/source/FindShowsMultipleArtists.js

```diff
@@ -0,0 +1,219 @@
+
+// @license
+// Copyright (c) 2018 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+'use strict';
+
+const eventsService = 'https://app.ticketmaster.com/discovery/v2/events?apikey=6g6GruAsAGU9RyT3lfjUFYtLSnvUITDe&radius=1000';
+const geohashCharMap = '0123456789bcdefghjkmnpqrstuvwxyz';
+
+defineParticle(({DomParticle, html, log, _fetch}) => {
+  const host = `[find-shows]`;
+  const styles = html`
+  <style>
+    ${host} {
+      padding: 0 24px 24px;
+      overflow: auto;
+    }
+    ${host} [logo] {
+      font-weight: bold;
+      font-style: italic;
+    }
+    ${host} [list] {
+      margin: 0 -8px;
+    }
+    ${host} [buttons] {
+      display: flex;
+      padding-top: 8px;
+    }
+    ${host} [button] {
+      transition: all 0.3s ease-in-out;
+      border-radius: 24px;
+      border: 1px solid rgba(0,0,0,.1);
+      font-size: 14px;
+      font-weight: bold;
+      line-height: 40px;
+      text-align: center;
+      color: #444;
+      background: #eee;
+      margin-right: 16px;
+    }
+    ${host} [button] > div {
+      border-radius: 20px;
+      padding: 0 20px;
+      background-color: #fefefe;
+    }
+    ${host} [glowing] {
+      -webkit-animation-name: glowing;
+      -webkit-animation-duration: 1.8s;
+      -webkit-animation-timing-function: ease-in-out;
+      -webkit-animation-iteration-count: 2;
+    }
+    ${host} [glowing]:hover {
+      background: rgba(3,169,244,.3);
+      box-shadow: 0 0 20px rgba(3, 169, 244, .3);
+    }
+    @-webkit-keyframes glowing {
+      0% {
+        background: rgba(3,169,244,0);
+        box-shadow: 0 0 20px rgba(3, 169, 244, 0);
+      }
+      25% {
+        background: rgba(3,169,244,0.2);
+        box-shadow: 0 0 20px rgba(3, 169, 244, 0.2);
+      }
+      50% {
+        background: rgba(3,169,244,.3);
+        box-shadow: 0 0 20px rgba(3, 169, 244, .3);
+      }
+      75% {
+        background: rgba(3,169,244,0.2);
+        box-shadow: 0 0 20px rgba(3, 169, 244, 0.2);
+      }
+      100% {
+        background: rgba(3,169,244,0);
+        box-shadow: 0 0 20px rgba(3, 169, 244, 0);
+      }
+    }
+  </style>
+  `;
+
+  const template = html`
+<div find-shows>
+${styles}
+  <div logo>Ticketmaster</div>
+  <div list slotid="listing"></div>
+  <div buttons>
+    <div button on-click="moreShows"><div>More shows like this</div></div>
+    <div button><div glowing>Buy tickets</div></div>
+  </div>
+</div>
+  `;
+  return class extends DomParticle {
```

Comment:
Add a TODO to unify with the other fetcher? The other one could use this one with a collection of a single artist I think.